audius-cli

```diff
@@ -859,5 +908,52 @@ def launch_notifications(ctx):
     )
 
 
+@cli.command()
+@click.pass_context
+@click.argument("name", required=True)
+def plugin_register(ctx, name):
+    print(f"registering plugin {name}")
+    override_env = get_override_path(ctx, "discovery-provider")
+    plugins = dotenv.get_key(override_env, REGISTERED_PLUGINS)
+    if plugins is not None:
+        plugins = plugins.split(",")
+    else:
+        plugins = []
+    plugins.append(name)
+    plugins = ",".join(plugins)
+    dotenv.set_key(override_env, REGISTERED_PLUGINS, plugins)
+    return
+
+
+@cli.command()
+@click.pass_context
+@click.argument("name", required=True)
+def plugin_deregister(ctx, name):
+    print(f"deregistering plugin {name}")
+    override_env = get_override_path(ctx, "discovery-provider")
+    plugins = dotenv.get_key(override_env, REGISTERED_PLUGINS)
+    if plugins is not None:
+        plugins = plugins.split(",")
+    else:
+        return []
+    if name in plugins:
+        plugins.remove(name)
+        # last plugin removed
+        if not plugins:
+            return []
+    plugins = ",".join(plugins)
+    dotenv.set_key(override_env, REGISTERED_PLUGINS, plugins)
+    return
+
+
+@cli.command()
+@click.pass_context
+def plugins(ctx):
```

Comment:
i can't tell if its more clear to rename to `print_plugins` ?? ill leave up to you

Reply:
ah yeah i like that!